model_bakery/utils.py

```diff
@@ -6,6 +6,8 @@
 from types import ModuleType
 from typing import Any, Callable, Optional, Union
 
+from django.apps import apps
+
```

Comment:
This is a tricky one, but I'll try to explain:

The `setup.py` file imports the `model_bakery` module just to center the project's version, author name etc in the module's root. But, [this very old commit](https://github.com/berinhard/model_mommy/commit/2c7640eccf0513d8c1d2f4267a72075c6920acfe#diff-38f8d480ffa2074444b43a4581793d024215389b0579376e42aa383d3d4454bc) (still in model mommy) introduces a shortcut to import the sequence function as `from model_bakery import seq` instead of `from model_bakery.recipes import seq`. So, whenever an user tries to install model bakery using `pip` or by manually running `python setup.py install`, this function `seq` gets imported. And this is the base for the errors we're seeing.

The `seq` function was moved to this `utils.py` and it wasn't a problem until today because it only depends on Python libs. That's why people can run the install without problems. But, now we're importing `django` here, people have to have Django installed **before** model bakery so the `setup.py` doesn't raise an import error. Sad but true.